_posts/2024-02-07-From-Small-to-Tall.html

```diff
@@ -12,7 +12,7 @@ <h2>From Small to Tall</h2>
 
 
 <div style="text-align: center">
-  <img src="/images/Blog Images/SmalltoTall.png"  class="size-small" style=" padding: 10px;" </img>
+  <img src="/images/Blog Images/SmalltoTall.png"  class="size-small" style="height: 30%; width: auto; padding: 10px;" </img>
```

Comment:
you never want to have css within your html - can you move these styles to a css class? If it makes sense to add it to the .size-small class you could do that, or make a new class just for styling images.